<launch>
  <node pkg="rviz2" exec="rviz2"
        args="-d $(find-pkg-share img_transform)/config/transform.rviz"
        namespace="global"/>

  <node pkg="tf2_ros" exec="static_transform_publisher"
        args="--frame-id world --child-frame-id previous" />

  <node pkg="img_transform" exec="img_transform">
  </node>

  <node pkg="teleop_twist_keyboard" exec="teleop_twist_keyboard" name="teleop" output="screen" launch-prefix="xterm -e">
  </node>

  <node pkg="img_transform" exec="se_sync">
  </node>
  
  <!-- <node pkg="img_transform" exec="wheel_encoder">
  <param from="$(find-pkg-share nuturtle_description)/config/diff_params.yaml"/>
  </node> -->

</launch>